src/status_im/contexts/chat/messenger/composer/style.cljs

```diff
@@ -95,17 +95,16 @@
     :background-color colors/neutral-95-opa-70}))
 
 (defn blur-container
-  [height focused?]
-  (reanimated/apply-animations-to-style
-   {:height height}
+  [composer-default-height {:keys [blur-container-elevation]}]
+  [{:elevation blur-container-elevation}
    {:position                :absolute
-    :elevation               (if-not @focused? 10 0)
```

Comment:
Are we removing the shadow completely on Android?

Reply:
Nope, moved to `blur-container-elevation`